actionpack/lib/action_controller/railtie.rb

```diff
@@ -67,5 +67,12 @@ class Railtie < Rails::Railtie #:nodoc:
         config.compile_methods! if config.respond_to?(:compile_methods!)
       end
     end
+
+    initializer "setup session store" do |app|
+      ActiveSupport.on_load(:action_controller) do
+        app_name = app.class.name ? app.railtie_name.sub(/_application$/, '') : ''
```

Comment:
@kaspth The problem was with all the tests using `make_basic_app` helper because of this line - https://github.com/prathamesh-sonpatki/rails/blob/f8244070e8098d930377d3ef99bf3d44fa7d4e83/railties/test/isolation/abstract_unit.rb#L172